packages/flutter/lib/src/widgets/form.dart

```diff
@@ -465,6 +464,24 @@ class FormField<T> extends StatefulWidget {
   /// [FormState.save].
   final FormFieldSetter<T>? onSaved;
 
+  /// An optional property that forces the [FormFieldState] into an error state
+  /// by directly setting the [FormFieldState.errorText] property without
+  /// running the validator function.
+  ///
+  /// When the [forceErrorText] property is provided, the [FormFieldState.errorText]
+  /// will be set to the provided value, causing the form field to be considered
+  /// invalid and to display the error message specified.
+  ///
+  /// When [validator] is provided, [forceErrorText] will override any error that it
+  /// returns. [validator] will not be called unless [forceErrorText] is null.
+  ///
+  /// See also:
+  ///
+  /// * [InputDecoration.errorText], which is used to display error messages in the text
```

Comment:
What do you mean here by not effecting the field's state? I think explicitly setting `InputDecoration.errorText` changes the fields state to an error state right?

Reply:
`InputDecoration.errorText` are just for the `InputDecorator` widget and does not effect the `FormField` state. So calling `FormState.validate()` will return `true` even when we set the `InputDecoration.errorText` to a non-null value.

Reply:
I see what you mean, thank you for the clarification.

Reply:
of course.